client/menu/menu.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+    <link rel="stylesheet" href="../../third_party/codemirror-5.19.0/lib/material.css">
+    <link rel="stylesheet" href="menu.css">
+    <script src="../../third_party/angular-1.6.1/angular.min.js"></script>
+  </head>
+  <body>
+    <div class="tie-menu-question-list-wrapper">
```

Comment:
Ah, so, when I talked about a dummy menu page, I just meant something that has a text/pointer that redirects to our current question page -- but the whole thing needs to still be fully usable; this is no longer just a development prototype. People are downloading and using the current application, so every change to it needs to result in a coherent UX.

You could perhaps have a single card that advertises a question set on strings, or something similar. But the data needs to be real and the UX resulting from this change needs to make sense.